api/tests/routes/native/v1/offerers_test.py

```diff
@@ -180,3 +180,11 @@ def test_get_venue_always_has_banner_url(self, client):
             assert response.status_code == 200
 
         assert response.json["bannerUrl"] is not None
+
+    def test_venue_supports_phone_numbers(self, client):
+        invalid_phone_number = "+33594282769"
```

Comment:
```suggestion
        invalid_phone_number = "+33123456789"
```
montre plus clairement qu'il manque un chiffre

Reply:
Okay 
j'ai pris un vrai exemple de notre bdd, s'pour ça

Je test ta proposition 👍